staging/src/k8s.io/component-base/metrics/clientgo/leaderelection/metrics.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package leaderelection
```

Comment:
would expect prometheus in the package path.

Am not sure about `clientgo/`. I tend to prefer `client/` only, but am not even that we need the `clientgo/` at all, at least for leader election.

Reply:
appended prometheus to directory path.